app/org/sagebionetworks/bridge/models/healthdata/RecordExportStatusRequest.java

```diff
@@ -0,0 +1,24 @@
+package org.sagebionetworks.bridge.models.healthdata;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.sagebionetworks.bridge.json.JsonUtils;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+/**
+ * POJO to wrap synapse exporter status infomation, including a list of record ids and a exporter status
+ */
+public class RecordExportStatusRequest {
```

Comment:
Shouldn't have verbs in entity names or any references to HTTP (request/response). So this is ExportStatuses, ExportJobStatus, or something similar.